pkgs/http_profile/README.md

```diff
@@ -1,2 +1,68 @@
-An **experimental** package that allows HTTP clients outside of the Dart SDK
-to integrate with the DevTools Network tab.
+[![pub package](https://img.shields.io/pub/v/http_profile.svg)](https://pub.dev/packages/http_profile)
+[![package publisher](https://img.shields.io/pub/publisher/http_profile.svg)](https://pub.dev/packages/http_profile/publisher)
+
+A package that allows HTTP clients outside of the Dart SDK to integrate with
+the DevTools Network tab.
+
+## Using
+
+`HttpClientRequestProfile.profile` returns an `HttpClientRequestProfile` object.
+Populating the fields of that object with information about an HTTP request and
+about the response to that request will make that information show up in the
+DevTools Network tab.
+
+```dart
+import 'package:http_profile/http_profile.dart';
+
+void main() {
```

Comment:
Maybe make this look more realistic by calling this function something like:

```dart
Future<String> get(Uri uri) {
  final profile = HttpClientRequestProfile.profile(
    requestStartTime: DateTime.now(),
    requestMethod: 'GET',
    requestUri: uri.toString(),
  );
  profile?.connectionInfo = {
      'localPort': 1285,
      'remotePort': 443,
      'connectionPoolId': '21x23',
    };

  profile.requestData.proxyDetails = HttpProfileProxyData(
    host: 'https://www.example.com',
    username: 'abc123',
    isDirect: true,
    port: 4321,
  );

  // Make the HTTP request and populate the response data.

  profile.requestData.proxyDetails = HttpProfileProxyData(
    host: 'https://www.example.com',
    username: 'abc123',
    isDirect: true,
    port: 4321,
  );
  profile.responseData.headersListValues = {
    'connection': ['keep-alive'],
    'cache-control': ['max-age=43200'],
    'content-type': ['application/json', 'charset=utf-8'],
  };

  return responseString;
}
```
  ...

}```

Reply:
Done